pkg/kv/kvserver/protectedts/ptpb/protectedts.proto

```diff
@@ -89,28 +89,6 @@ message Metadata {
 
 // Record corresponds to a protected timestamp.
 message Record {
-  message SchemaObjectsTarget {
```

Comment:
is it safe to change proto like this? we have serialized records stored in protected keys, don't we?

Reply:
These fields were added only a couple days ago in https://github.com/cockroachdb/cockroach/pull/74211, since this is unreleased code I think it is safe?

Reply:
If nobody generated and serialized those, then it's fine (i'm thinking roachtests).